json-guides/microprofile-config.json

```diff
@@ -15,6 +15,65 @@
             ]
         },
         {
+            "name": "ConfigureViaInject",
+            "title": "Configuring ConfigProperties through Injection",
```

Comment:
I will let Denise change your title or you can change it to "Configuring with Default Injection Value"